application/src/main/resources/db/V11__Add_Help_Channel_Plus_Creator.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE add_help_channel
+(
+    channel_id BIGINT    NOT NULL PRIMARY KEY,
+    user_id    BIGINT    NOT NULL,
```

Comment:
user is a bad name, it would be better to give the context of who the user is. i.e. `author`